textile/chat-features.textile

```diff
@@ -391,8 +391,8 @@ Presence allows chat room users to indicate to others that they're online, as we
 ** @(CHA-PR7c)@ @[Testable]@ A subscription to presence may be removed, after which it shall receive no further events.
 * @(CHA-PR8)@ @[Testable]@ Users may subscribe to discontinuity events to know when there's been a break in presence. Their listener will be called when a discontinuity event is triggered from the room lifecycle. For presence, there shouldn't need to be user action as the underlying core SDK will heal the presence set.
 * @(CHA-PR9)@ Users may configure their presence options via the @RoomOptions@ provided at room configuration time.
-** @(CHA-PR9a)@ @[Testable]@ Setting @enter@ to false prevents the user from entering presence by means of the @ChannelMode@ on the underlying realtime channel. Entering presence will result in an error. The default is true.
-** @(CHA-PR9b)@ @[Testable]@ Setting @subscribe@ to false prevents the user from subscribing to presence by means of the @ChannelMode@ on the underlying realtime channel. This does not prevent them from receiving their own presence messages, but they will not receive them from others. The default is true.
+** @(CHA-PR9a)@ @[Testable]@ If @enter@ is set to true (or left unspecified), then the presence feature must set the @PRESENCE@ mode on the underlying Realtime channel.
```

Comment:
Also, any combination of `enter` and `subscribe` is permitted right? I mean do need to have validation for the same like we do in case of negative typing timeout

Reply:
In JS, it's if you leave it as `undefined` - at which point it takes the default value.

Reply:
> Also, any combination of `enter` and `subscribe` is permitted right? I mean do need to have validation for the same like we do in case of negative typing timeout

Any combination is permitted, yes.

Reply:
`default` is true for both right

Reply:
[kotlin-presence-options](https://github.com/ably-labs/ably-chat-kotlin/blob/45c7156e9d752d57cda379d0176d624c043f1d13/chat-android/src/main/java/com/ably/chat/RoomOptions.kt#L36)

Reply:
Yes - for some reason my IDE didn't commit the line where I added "default is true" will do that as a separate PR now.